gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -110,31 +110,29 @@ nnstreamer_query_connection_get_port (query_connection_handle connection)
   return conn->port;
 }
 
+
 /**
- * @brief Create requested socket.
+ * @brief Get socket address
  */
 static gboolean
-gst_tensor_query_socket_new (query_connection_handle conn_h,
-    GSocketAddress ** saddr)
+gst_tensor_query_get_saddr (const char *host, uint16_t port,
```

Comment:
Better to use `guint16` instead of `uint16_t`?

Reply:
Updated.

Reply:
Actually, as `TensorQueryConnection` has declared `port` as `guint16_t`, it's better to unify the type :) `uint16_t port` still exist.

Reply:
Changed GLib types to standard types. Please check it again.